### ISSUE TITLE
Can't access Gmail account because "This browser or app may not be secure"

### ISSUE THREAD

**Issue by SquareBottle** [TheGoddessInari/hamsket#318]
### Steps to reproduce

1. Open Hamsket
2. Try to go to Gmail account (or add a new one)
3. Try to log in
4. Get the "Couldn't sign you in: This browser or app may not be secure" error

### Expected behavior

Expected it to log in like normal.

### Actual behavior

Can't log in.

### ENVIRONMENT

> Hamsket: v0.6.3

> OS: 12.2.1

> Arch: x64 (Apple M1)

> BuildVersion: aeb5a15 (grafted, HEAD, tag: nightly, origin/aeb5a154d4931b0d921ce4e5855d87adeca5cebd) Tue, 1 Mar 2022 16:06:12 -0800

> Does this happen upstream: yes, no, unknown, or N/A

N/A

**Comment by TheGoddessInari** [comment]
How strange. That's what 026a1958 was supposed to fix. 

There "shouldn't" be a way for it to race and fail to attach anymore, but I'm seeing exactly that on occasion.

I checked the Network tab locally, and I'm seeing requests for accounts.google.com properly getting the User Agent.

I double checked, and this is still what everything else is utilizing as a workaround.

**Comment by ikenichiro** [comment]
If you use gmail language setting other than English, try re-add service and set the language to English before you attempt a login.
I don't know why, but that worked for me as a workaround.

**Comment by flyingdutchman** [comment]
Indeed, I changed the language of the login page from "English (United States)" to "English (United Kingdoms)" and it worked like a charm !

**Comment by k0d3r1s** [comment]
still happening, language change doesn't help. v0.6.3

**Comment by TheGoddessInari** [comment]
cee2f087 on the `testing` branch *might* fix this. I might upload a one-off pack to dropbox or something for Windows/Linux users who don't want to build themselves.
